fe/fe-core/src/main/java/com/starrocks/planner/PlanNode.java

```diff
@@ -498,7 +499,7 @@ public TPlan treeToThrift() {
         return result;
     }
 
-    protected void toNormalForm(TNormalPlanNode planNode, FragmentNormalizer normalizer){
+    protected void toNormalForm(TNormalPlanNode planNode, FragmentNormalizer normalizer) {
```

Comment:
why code-format changed?

Reply:
it seems we don't have a check-style rule for this